modules/brittbot/karma.py

```diff
@@ -72,8 +73,8 @@ def karma_award(jenni, msg):
             saying = random.choice(positive_sayings)
         else:
             saying = random.choice(negitive_sayings)
-        jenni.say("%s %s (Karma: %s)" % (
-            item, saying, jenni.brain['karma'][item]
+        jenni.say("%s (Karma: %s)" % (
+            saying % item, jenni.brain['karma'][item]
```

Comment:
Any chance we can use `.format()` here instead?
`%` for string interpolation is deprecated :(